src/nvim/highlight_defs.h

```diff
@@ -146,6 +147,7 @@ EXTERN const char *hlf_names[] INIT(= {
   [HLF_INACTIVE] = "NormalNC",
   [HLF_MSGSEP] = "MsgSeparator",
   [HLF_NFLOAT] = "NormalFloat",
+  [HLF_MSG] = "Message",
```

Comment:
how about `MsgArea`, for parallel with `MsgSeparator`/`ErrorMsg`/etc.

Reply:
`MsgArea` sounds good :+1: